.github/workflows/CoverageCollection.yml

```diff
@@ -35,10 +35,15 @@ on:
         required: false
         default: '-r tests/requirements.txt'
         type: string
+      test_directory:
+        description: 'Path to the directory containing tests (test working directory).'
+        required: false
+        default: 'tests'
+        type: string
       unittest_directory:
-        description: 'Path to the directory containing unit tests.'
+        description: 'Path to the directory containing unit tests (from test_directory).'
```

Comment:
I'd say "relative to" rather than "from". However, does it need to be relative? It seems that `test_directory` is not the path containing the tests, but the working dir where to execute them from. That unittest_directory is the actual location of the tests.

Reply:
Updated in b0f31605.